hand_tracking.py

```diff
@@ -0,0 +1,42 @@
+import cv2 as cv
+import mediapipe as mp
+
+class handDetector():
+    def __init__(self, mode=False, maxHands=2, modelComplexity=1, detectionCon=0.5, trackCon=0.5):
+        self.mode               = mode
+        self.maxHands           = maxHands
+        self.modelComplexity    = modelComplexity
+        self.detectionCon       = detectionCon
+        self.trackCon           = trackCon
+        self.mpHands            = mp.solutions.hands
+        self.hands              = self.mpHands.Hands(self.mode, self.maxHands, self.modelComplexity, self.detectionCon, self.trackCon)
+        self.mpDraw             = mp.solutions.drawing_utils
+
+    def findHands(self, img, draw=True):
+        imgRGB = cv.cvtColor(img, cv.COLOR_BGR2RGB)
+        self.results = self.hands.process(imgRGB)
+
+        if self.results.multi_hand_landmarks:
+            for handLms in self.results.multi_hand_landmarks:
+                if draw: self.mpDraw.draw_landmarks(img, handLms, self.mpHands.HAND_CONNECTIONS)
+
+        return img
+
+    def findPosition(self, img, handNo=0, draw=True):
+        lmList = []
+
+        if self.results.multi_hand_landmarks:
+            myHand = self.results.multi_hand_landmarks[handNo]
+
+            # Hand Labeling
+            for id, hand_handedness in enumerate(self.results.multi_handedness):
+                self.label = hand_handedness.classification[0].label
+
+            for id, lm in enumerate(myHand.landmark):
+                h, w, c = img.shape
+                cx, cy = int(lm.x * w), int(lm.y * h)
+                lmList.append([id, cx, cy])
```

Comment:
what this vars mean?
id, lm, h, w, c, cx, cy?
Could we rename this vars? Like, h = height

Reply:
All right, this is just the first version. I'll uptade with new features and corrections.

id - > landmark identifier 
lm -> hand landmarks
h -> height
c -> image shape
cx -> landmark position in X
cy -> landmark position in Y

Reply:
References https://google.github.io/mediapipe/solutions/hands.html